docs/detections/machine-learning/machine-learning.asciidoc

```diff
@@ -66,6 +66,10 @@ Or
 * Your shipped data is ECS-compliant, and {kib} is configured with the shipped
 data's index patterns in *{kib}* -> *{stack-manage-app}* -> *Data Views*.
 
+Or
+
+* You install one or more of the Advanced Analytics Integrations (see following section).
```

Comment:
Hi @susan-shu-c does `Advanced Analytics` need to be capitalized here? Also, swapping `see` for `refer to` to be ADA-compliant. 
```suggestion
* You install one or more of the advanced analytics integrations (refer to the following section).
```

Reply:
That's a great point, thank you! re: `refer to` ❤️ 

We've been capitalizing Advanced Analytics, any thoughts @ajosh0504 ?

Reply:
Hi @jmikell821 we're going to keep this capitalized to be consistent with the terminology, so hope that works!

Reply:
Yeah let's keep it as `Advanced Analytics integrations` (integrations lowercase).

Reply:
Done, thank you!